src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerConfigOptions.cs

```diff
@@ -15,6 +15,8 @@ public abstract class AnalyzerConfigOptions
         /// </summary>
         public static StringComparer KeyComparer { get; } = AnalyzerConfig.Section.PropertiesKeyComparer;
 
+        internal static ImmutableDictionary<string, string> EmptyDictionary = ImmutableDictionary.Create<string, string>(KeyComparer);
```

Comment:
Let's just call this `Empty`

Reply:
@agocke I think I prefer the name as is; "Empty" to me would imply it returns a no-op implementation of AnalyzerConfigOptions. This is really is an empty dictionary and a special one at that.